README.rst

```diff
@@ -32,6 +32,16 @@ Installation
 See doc/install.txt in the source code archive.
 Python 2.7.2 or later is needed.
 
+
+Install using Docker
+---------------------
+You can try linkchecker in an isolated enviroment without changing any local files or have to install all the dependencies via a Docker Container:
+
+``docker pull ellerbrock/linkchecker``
+
+https://hub.docker.com/r/ellerbrock/link-checker/
```

Comment:
hi @anarcat,

basically all the docker stuff on docker hub is "unofficial".
real big projects get added to the docker library: https://hub.docker.com/explore/, but not sure what it takes to get there.

look for the bash-it project we handle this like that.
they have a [bash-it](https://github.com/Bash-it/bash-it) main repo and a [docker-bash-it](https://github.com/Bash-it/bash-it-docker) which i maintain for example. when i find better ways to do stuff i update this and add it to the docker stuff i maintain ...